src/bind/Toolchain.cxx

```diff
@@ -101,3 +107,32 @@ void Toolchain::set_free() {
         return Godot::print("Warning: BoardRunner queued to be freed while still building");
     queue_free();
 }
+
+Array Toolchain::find_compilers() {
+    Array result;
+
+    auto compilerDefault = make_ref<Toolchain::CompilerInformation>();
+    compilerDefault->name = "Default";
+    compilerDefault->path = "-";
+    compilerDefault->version = "-";
```

Comment:
Would be good to see an alternative way to represent "just figure it out", if there are no compilers then that would be a real problem. Putting Default would suggest there is a compiler when there actually isnt one

Reply:
The responsibility for default compilers is now outside the bind class and the default compiler option is only shown, when there is actually a compiler.